internal/blockutils/cmd.go

```diff
@@ -0,0 +1,31 @@
+package blockutils
+
+import (
+	"bufio"
+	"gobar/internal/log"
+	"os/exec"
+)
+
+// RunCmdStdout runs a command, then captures and returns the output
+func RunCmdStdout(cmd *exec.Cmd) ([]string, error) {
+	var stdoutLines []string
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return nil, err
+	}
+	defer stdout.Close()
+	if err := cmd.Start(); err != nil {
+		log.FileLog(err)
+		return nil, err
+	}
+
+	sc := bufio.NewScanner(stdout)
+
+	for sc.Scan() {
+		stdoutLines = append(stdoutLines, sc.Text())
+	}
+
```

Comment:
don't forget to check the error. it's worth logging the error (at least once - i like to avoid repeating an error endlessly, but it should still be in your logs)